### ISSUE TITLE
Audio player in VK.com doesn't work

### ISSUE THREAD

**Issue by qqew** [Feodor2/Mypal#31]
It seems that VK.com recently redesigned something so Mypal as well as Firefox ESR cannot play audio on It. Could be great to fix this problem.

**Comment by Feodor2** [comment]
Не подтверждаю - работает аудио у меня. Проверь не блокируется ли vkuseraudio.net контактовая музыка через него идёт.

*Предположу что руский понимаеш.

**Comment by qqew** [comment]
Да, косяк мой. Причина в расширении uBlock Origin. Странно, но раньше оно не мешало.
Sorry my bad. The reason is the extension uBlock Origin, which previously had no effect.

**Comment by dalxlab** [comment]
аналогичная беда. версия - последняя. флэш (хоть это и не суть) - обновлён. кодеки - стоят. плагинов - нет (только изменил тему). но - музыка не играет. бесконечно крутит прогресс загрузки и - ничего.
обратил внимание, что такая беда - с музыкой, что отдаётся в формате m3u8. то что в mp3 - играется без проюлем. реклама ;) - играется бодро. если "кинуть" в Media Player Classic эту ссылку - играет корректно...
может что-то всё же в настройках может повлиять? или кодеки иные нужны? или...?

Windows XP sp3, K-Lite (последний для XP), MyPal - последний.

**Comment by Feodor2** [comment]
m3u8 вообще в пельмене работает
Мне нужен образец такой музыки с контакта.

**Comment by dalxlab** [comment]
ссылка для примера:
https://vk.com/wall-54896111_65994
то, что пытается открыть (хотя думаю, что это от раза к разу меняется):
https://cs1-46v4.vkuseraudio.net/p14/0cfe6ab068/0e93e32058b7b9/index.m3u8
сам файл m3u8:
[index.zip](https://github.com/Feodor2/Mypal/files/2923763/index.zip)

**Comment by qqew** [comment]
После более пристального внимания вынужден признать, что блокировщик рекламы совершенно не при чем. Проблема проявляется спонтанно с непонятной закономерностью. Просто одни треки ВК играет, другие тупо висят и не воспроизводятся. (XP sp-3, РМ последний, в FF ESR картина та же!)

**Comment by dalxlab** [comment]
обратил внимание, что в новом/текущем огнелисе (ставил давеча свежачок на иной комп) при проигрывании той же композиции - вконтакт отдаёт не странного вида m3u8, а как и раньше - mp3.
_(m3u8 и впрямь странный. нарезка кусков композиции. и куски поочереди то шифрованны, то нет. возможно это иль что-то подобное и смущает MyPal)._ 
предположил, что дело в юзерагенте. вероятно вконтакт по каким-то своим причинам что-то там решает и отдаёт музыку в таком виде. и правда. перебил юзерагента для вконтакта на огнелиса 64-го и винду 7-ю и вуаля - всё шуршит. вконтакт отдаёт стандартную mp3-ку, а браузер её спокойно играет... 
...
удалил настроку. переоткрыл страницу - снова играет. непонятно :))). могу лишь предположить, что вконтакт какое-то время хранит настройки сессии. пока оставил для вконтакта юсерагента "нового". погляжу как оно будет.
...

**Comment by Feodor2** [comment]
по ссылке vk.com играет нормально 
по следующей ссылке 404 - она выдается на один раз.
вообще что бы я не включал всё работает,  ставил разные юзер агенты.

**Comment by qqew** [comment]
Странным образом вдруг стало всё воспроизводиться. Кэеш не чистил, агента не менял. Просто примерно после мартовских выходных всё починилось само собой и в РМ и в FF. Все-таки, похоже, к этому причастен сам ВК.
